src/imars3d/ui/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.2.0.dev165"
```

Comment:
This file should be removed and marked in the `.gitignore`

Reply:
file is removed